requirements.txt

```diff
@@ -0,0 +1,5 @@
+mock==1.0.1
+pep8==1.5.7
+pytest==2.6.4
+requests==2.5.0
+git+https://github.com/soulnai/gaeenv.git@develop#egg=gaeenv
```

Comment:
Nice idea. I will change .travis.yml in my fork.

2015-01-16 17:13 GMT+02:00 Lance Linder <notifications@github.com>:

> In requirements.txt
> <https://github.com/llinder/gaeenv/pull/3#discussion-diff-23085243>:
>
> > @@ -0,0 +1,5 @@
> > +mock==1.0.1
> > +pep8==1.5.7
> > +pytest==2.6.4
> > +requests==2.5.0
> > +git+https://github.com/soulnai/gaeenv.git@develop#egg=gaeenv
>
> probably would make sense to install this from the .travis.yml file
> instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/llinder/gaeenv/pull/3/files#r23085243>.
>